http-add-on/templates/svc.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
+    httpscaledobjects.http.keda.sh/version: {{ .Values.version }}
+    keda.sh/addon: keda-http-addon
+  name: keda-http-addon-controller-manager-metrics-service
+  namespace: {{ .Release.Namespace }}
+spec:
+  ports:
+  - name: https
+    port: 8443
```

Comment:
Let's open this up through the values, but a dedicated issue to track this is OK.

Reply:
Did on b3ce426